src/sonic-py-common/sonic_py_common/daemon_base.py

```diff
@@ -32,12 +36,16 @@ def db_connect(db_name, namespace=EMPTY_NAMESPACE):
 
 
 class DaemonBase(Logger):
-    def __init__(self, log_identifier):
-        super(DaemonBase, self).__init__(
-            log_identifier=log_identifier,
-            log_facility=Logger.LOG_FACILITY_DAEMON,
-            log_option=(Logger.LOG_OPTION_NDELAY | Logger.LOG_OPTION_PID)
-        )
+    def __init__(self, log_identifier, use_syslogger=False):
```

Comment:
@xincunli-sonic @nazariig can we have the default `use_syslogger`=True

Reply:
@prgeor totally agree. Better to force all the daemons/libraries to use a new implementation by default